cas-server-documentation/installation/OAuth-OpenId-Authentication.md

```diff
@@ -95,15 +28,18 @@ Every OAuth client must be defined as a CAS service (notice the new *clientId* a
 <bean id="serviceRegistryDao" class="org.jasig.cas.services.InMemoryServiceRegistryDaoImpl">
```

Comment:
Good point, yes. I'll create an issue to track this. Thanks for bringing it up. It is confusing right now.